README.md

```diff
@@ -1,58 +1,19 @@
-# OpenShift Console Plugin Template
-
-This project is a minimal template for writing a new OpenShift Console dynamic
-plugin.
-
-[Dynamic plugins](https://github.com/openshift/console/tree/master/frontend/packages/console-dynamic-plugin-sdk)
-allow you to extend the
-[OpenShift UI](https://github.com/openshift/console)
-at runtime, adding custom pages and other extensions. They are based on
-[webpack module federation](https://webpack.js.org/concepts/module-federation/).
-Plugins are registered with console using the `ConsolePlugin` custom resource
-and enabled in the console operator config by a cluster administrator.
-
-Using the latest `v1` API version of `ConsolePlugin` CRD, requires OpenShift 4.12
-and higher. For using old `v1alpha1` API version us OpenShift version 4.10 or 4.11.
-
-For an example of a plugin that works with OpenShift 4.11, see the `release-4.11` branch.
-For a plugin that works with OpenShift 4.10, see the `release-4.10` branch.
-
-[Node.js](https://nodejs.org/en/) and [yarn](https://yarnpkg.com) are required
-to build and run the example. To run OpenShift console in a container, either
-[Docker](https://www.docker.com) or [podman 3.2.0+](https://podman.io) and
-[oc](https://console.redhat.com/openshift/downloads) are required.
-
-## Getting started
-
-After cloning this repo, you should update the plugin metadata such as the
-plugin name in the `consolePlugin` declaration of [package.json](package.json).
-
-```json
-"consolePlugin": {
-  "name": "console-plugin-template",
-  "version": "0.0.1",
-  "displayName": "My Plugin",
-  "description": "Enjoy this shiny, new console plugin!",
-  "exposedModules": {
-    "ExamplePage": "./components/ExamplePage"
-  },
-  "dependencies": {
-    "@console/pluginAPI": "*"
-  }
-}
-```
+# OpenShift Console Plugin for NFS PVC Resources
+
+This project is an OpenShift Console dynamic plugin that extends the OpenShift UI with features to manage **NFS PVC** resources. The plugin introduces a new section in the dashboard called **Storage Automations**, where users can view a list of existing NFS PVCs and create new ones via a form.
```

Comment:
This project is an OpenShift Console dynamic plugin that extends the OpenShift UI with features to manage [`nfspvc.dana.io` resources](https://github.com/dana-team/nfspvc-operator). The plugin introduces a new section in the UI called `Storage Automations`, where users can view a list of existing `NFSPVCs` and create new ones via a form.